api/app/tests/hfi/test_hfi_endpoint_response_loaded.json

```diff
@@ -11,8 +13,192 @@
       }
     ]
   },
-  "selected_station_code_ids": [230],
   "selected_fire_center_id": 1,
-  "planning_area_hfi_results": [],
-  "planning_area_fire_starts": {}
-}
+  "planning_area_hfi_results": [
+    {
+      "planning_area_id": 1,
+      "all_dailies_valid": true,
+      "highest_daily_intensity_group": null,
+      "mean_prep_level": null,
+      "daily_results": [
+        {
+          "date": "2020-05-21",
+          "dailies": [],
+          "fire_starts": {
+            "label": "0-1",
+            "value": 1,
+            "lookup_table": {
+              "1": 1,
+              "2": 1,
+              "3": 2,
+              "4": 3,
+              "5": 4
+            }
+          },
+          "mean_intensity_group": null,
+          "prep_level": null
+        },
+        {
+          "date": "2020-05-22",
+          "dailies": [],
+          "fire_starts": {
+            "label": "0-1",
+            "value": 1,
+            "lookup_table": {
+              "1": 1,
+              "2": 1,
+              "3": 2,
+              "4": 3,
+              "5": 4
+            }
+          },
+          "mean_intensity_group": null,
+          "prep_level": null
+        },
+        {
+          "date": "2020-05-23",
+          "dailies": [
+            {
+              "daily": {
+                "code": 230,
+                "status": "FORECAST",
+                "temperature": 23.0,
+                "relative_humidity": 36.0,
+                "wind_speed": 7.0,
+                "wind_direction": 300.0,
+                "grass_cure_percentage": null,
+                "precipitation": 1.8,
+                "ffmc": 84.294,
+                "dmc": 79.937,
+                "dc": 463.498,
+                "isi": 2.724,
+                "bui": 111.71,
+                "fwi": 12.966,
+                "danger_class": null,
+                "observation_valid": true,
+                "observation_valid_comment": null,
+                "rate_of_spread": 0.18030340382396534,
+                "hfi": 182.97053672131358,
+                "intensity_group": 1,
+                "sixty_minute_fire_size": 0.006935706342019749,
+                "fire_type": "SUR",
+                "error": true,
+                "error_message": null,
+                "date": "2020-05-23T20:00:00+00:00",
```

Comment:
Should these `date` fields have timezones attached?

Reply:
I believe so, we're passing down what we're getting from wf1, and it's relevant that the forecast is for a specific time of day.